docs/manual/java/releases/Highlights15.md

```diff
@@ -0,0 +1,29 @@
+# What's new in Lagom 1.5?
+
+This page highlights the new features of Lagom 1.5. If you want to learn about the changes you need to make when you migrate to Lagom 1.5, check out the [[Lagom 1.5 Migration Guide|Migration15]].
+
+
+
+## TLS Support
+
+Lagom in both [[dev mode|ConfiguringServicesInDevelopment#Using-HTTPS-in-development-mode]] and [[tests|Test#How-to-use-TLS-on-tests]] supports basic usage of TLS by means of self-signed certificates provided by the framework.
```

Comment:
The different page name of `Test` in `java` vs `TestingServices` in `scala` is the only difference between the `Highlights15.md` files.

Just because of this link I had to duplicate `Highlights15.md`.

Reply:
That sucks.  Is this easily fixable?

Reply:
Only option is to remove the link.

Reply:
This is a page we'll barely ever come back to.

Reply:
Instead of duplicating the highlights file, I would prefer to do something like: "see the docs for Java or Scala" and have different links. Is this possible?

Reply:
This is not possible.

I think the `Highlights15.md` is a page we won't be editing anymore. We already have a ton of duplicated pages that need more maintenance. Once `1.5.0` is out, `Highlights15.md` will very rarely be edited.

Reply:
Implemented in https://github.com/lagom/lagom/pull/1788.